docs/methods/org.update.md

```diff
@@ -0,0 +1,69 @@
+# org.update
+
+`POST /api/v1/org.update`
+
+## Description
+
+Updates the designated org with the specified properties.
+
+---
+
+## Auth logic
+
+Requestor must be authenticated and assigned with the `yeep.org.write` permission for the designated org scope or the _global_ scope.
+
+## Parameters
+
+### Body
+
+- **id** _(string)_ — org ID (required)
+- **name** _(string)_ — the name of the organization (required)
+- **slug** _(string)_ — the URL key of the organization, a.k.a. slug (required)
+
+_Please note: one of `name` or `slug` must be specified, otherwise update makes no sense._
+
+---
+
+## Returns
+
+**200 OK** alongside `Object` with the following properties:
+
+- **ok** _(boolean)_ — indicates whether the request was successfully completed
+- **error** _(Object)_ — contains error details in case of an error
+- **role** _(Object)_ — role details
```

Comment:
Typo: `role` should be `org`. Please also make sure to update the description text accordingly.